hs_core/views/__init__.py

```diff
@@ -1091,14 +1091,27 @@ class AddUserForm(forms.Form):
         user = forms.ModelChoiceField(User.objects.filter(is_active=True).all(),
                                       widget=autocomplete_light.ChoiceWidget("UserAutocomplete"))
 
+    class AddUserContriForm(forms.Form):
+        user = forms.ModelChoiceField(User.objects.filter(is_active=True).all(),
+                                      widget=autocomplete_light.ChoiceWidget("UserAutocomplete", attrs={'id':'contri'}))
+
+    class AddUserInviteForm(forms.Form):
+        user = forms.ModelChoiceField(User.objects.filter(is_active=True).all(),
+                                      widget=autocomplete_light.ChoiceWidget("UserAutocomplete", attrs={'id':'invite'}))
+
+    class AddUserHSForm(forms.Form):
+        user = forms.ModelChoiceField(User.objects.filter(is_active=True).all(),
+                                      widget=autocomplete_light.ChoiceWidget("UserAutocomplete", attrs={'id':'hs-user'}))
+
     class AddGroupForm(forms.Form):
         group = forms.ModelChoiceField(Group.objects.filter(gaccess__active=True).exclude(name='Hydroshare Author').all(),
                                        widget=autocomplete_light.ChoiceWidget("GroupAutocomplete"))
 
     return {
-        'add_owner_user_form': AddUserForm(),
+        'add_view_contri_user_form': AddUserContriForm(),
+        'add_view_invite_user_form': AddUserInviteForm(),
+        'add_view_hs_user_form': AddUserHSForm(),
         'add_view_user_form': AddUserForm(),
-        'add_edit_user_form': AddUserForm(),
         'add_view_group_form': AddGroupForm(),
```

Comment:
Do we need to do he same for AddGroupForm? or is that ok?

Reply:
good catch. Currently there is only one active use of add_view_group_form. So we don't have dup id coming out yet. But if someone reuse the same template, there will be duplicate ids.

Reply:
Leave a comment there and let people know of the risk?

Reply:
Please add a comment at this point in the code so future devs know of the danger